test/unit/config.test.js

```diff
@@ -0,0 +1,102 @@
+const globals = require(`${__dirname}/../../app/globals`) // Always required
+const config = require(`${__dirname}/../../app/config`)
+
+describe('Config', () => {
+  it ('should export object', () => {
+    expect(config).toBeInstanceOf(Object)
+  })
+  describe('#getFrontendProps()', () => {
+    const input = {
+      apis: {
+        availableInFrontend: true,
+        doc: 'Connected APIs',
+        format: Array,
+        default: [],
+        publishId: {
+          availableInFrontend: true,
+          type: String,
+          default: ''
+        },
+        enabled: {
+          type: Boolean,
+          default: true
+        },
+        name: {
+          availableInFrontend: true,
+          format: String,
+          default: 'No Name'
+        },
+        host: {
+          availableInFrontend: true,
+          format: 'ipaddress',
+          default: '0.0.0.0'
+        },
+        port: {
+          availableInFrontend: true,
+          format: 'port',
+          default: 3000
+        },
+        database: {
+          availableInFrontend: true,
+          format: String,
+          default: ''
+        },
+        version: {
+          availableInFrontend: true,
+          format: String,
+          default: '1.0'
+        },
+        menu: {
+          availableInFrontend: true,
+          doc: 'Collection menu ordering and grouping',
+          format: Array,
+          default: []
+        },
+        credentials: {
+          format: Object,
+          clientId: {
+            format: String,
+            default: 'testClient'
+          },
+          secret: {
+            format: String,
+            default: 'superSecret'
+          }
+        }
+      }
+    }
+    const out = {
+      apis: {
+        publishId: true,
+        name: true,
+        host: true,
+        port: true,
+        database: true,
+        version: true,
+        menu: true
+      } 
+    }
+    it ('should return object', () => {
+      expect(config.getFrontendProps()).toBeInstanceOf(Object)
+    })
+    const schema = config.getFrontendProps(input, {})
```

Comment:
I'm just advising on best practices that we use in our test suites. Re-using an object in several tests is a very common pattern and `before`/`after` hooks are what we typically use.

Other examples:

- api-wrapper-core: https://github.com/dadi/api-wrapper-core/blob/master/test/unit/terminators.js#L61-L69

- API: https://github.com/dadi/api/blob/d8921e678ec427153e2f5d48d59897523e3c6cd1/test/unit/cache.js#L31-L42

- Web: https://github.com/dadi/web/blob/6c60c1fbc9604e53de6fed19df8de17d68f65e5f/test/unit/session.js#L35-L41

In any case, do as you prefer.